lib/mixins/error_notifier_methods.rb

```diff
@@ -5,8 +5,9 @@ module ErrorNotifierMethods
   include Logging
 
   def notify_admins_of_error(error, error_message, error_context)
-    Honeybadger.notify(error_class: error, error_message: error_message, context: error_context)
-    Logging.logger.error { error_message }
+    Logging.logger.error { error }
+    Logging.logger.error { [error_message, error_context] }
+    Honeybadger.notify(error, error_message: error_message, context: error_context)
```

Comment:
Should we add backtrace for further debugging info?

Reply:
Now that am passing the whole error object, I think it is unnecessary. I will check how the honeybadgers sees the backtrace and then can add it if required.